llpc/unittests/util/testPipelineDumper.cpp

```diff
@@ -351,6 +351,26 @@ TEST(PipelineDumperTest, TestReverseThreadGroupCompute) {
   HashModifiedFunc expectHashToBeEqual = [](const GenerateHashParams &params) { return false; };
   runComputePipelineVariations(modifyBuildInfo, expectHashToBeEqual);
 }
+#if VKI_RAY_TRACING
+// =====================================================================================================================
+// Test the internalRtShaders option.
+
+TEST(PipelineDumperTest, TestInternalRTShadersGraphics) {
+  ModifyGraphicsBuildInfo modifyBuildInfo = [](GraphicsPipelineBuildInfo *buildInfo) {
+    buildInfo->options.robustBufferAccess = true;
```

Comment:
These test use `robustBufferAccess`, should they use `isInternalRtShader` instead?

Reply:
Thanks for pointing this out.